src/options/view.rs

```diff
@@ -319,6 +319,9 @@ impl TimeFormat {
             "iso" => Ok(Self::ISOFormat),
             "long-iso" => Ok(Self::LongISO),
             "full-iso" => Ok(Self::FullISO),
+            fmt if fmt.starts_with('+') => Ok(Self::Custom {
+                fmt: fmt[1..].to_owned(),
+            }),
```

Comment:
I like the idea of being able to customise it more and everything, but at this point, and seeing how the  verification is done I think that the `+` is not mandatory and could even be thrown out, because we could say something along the line of: "Either one of those preset or your style" in terms of help and then here just verify the format is right. I didn't reallly looked more inside how the rustfmt works but I think this could be done, the only 'problem' of my solution seems to be a higher memory usage but for like one function call.

Reply:
I think the best argument for keeping the `+` prefix is that it's how I believe stuff like `date` does is. But I also see the value in not having it just for legacy's sake.

Reply:
Yes your right for the `date`